### PR TITLE
feat(FOROME-1555): Redesign of units list

### PR DESCRIPTION
**Pull request by AndreyUstyumenko**
https://quantori.atlassian.net/browse/FOROME-1555

### CODE REVIEW

File: src/components/units-list/units-list-unit/units-list-unit.module.css [hunk 1]
```diff
@@ -1,45 +1,51 @@
 .unit {
-  padding: 0 0 theme("spacing.2") theme("spacing.2");
+  padding: 6px 0;
+  padding-left: 14px;
+
+  &__selected {
+    border: 1px solid theme('colors.grey.dark');
```
Comment (EvgeniySobolev) [inline src/components/units-list/units-list-unit/units-list-unit.module.css]: When we select unit and this border appears we can see little jumps of neighboor units. Maybe you can use a transparent border for default state of units to solve this problem

File: src/components/units-list/units-list-unit/units-list-unit.module.css [hunk 2]
```diff
@@ -1,45 +1,51 @@
...
 .unit-title {
   display: flex;
   align-items: center;
 
   &__title {
+    @apply typography-s;
+
     margin-right: 4px;
-    color: theme("colors.black");
+    color: theme('colors.black');
```
Comment (EvgeniySobolev) [inline src/components/units-list/units-list-unit/units-list-unit.module.css]: Single quotes should have caused an Stylelint error. Check your precommit hooks please

Comment (AndreyUstyumenko) [reply src/components/units-list/units-list-unit/units-list-unit.module.css]: we have single quotes set for strings

File: src/components/units-list/units-list-unit/units-list-unit.module.css (hunk 1 above)
Comment (AndreyUstyumenko) [reply src/components/units-list/units-list-unit/units-list-unit.module.css]: yes, good point

**Review by Lapk1n — APPROVED** [review]

File: src/components/units-list/units-list-unit/units-list-unit.module.css (hunk 1 above)
Comment (Lapk1n) [reply src/components/units-list/units-list-unit/units-list-unit.module.css]: Decided to use an outline

**Review by EvgeniySobolev — APPROVED** [review]

**Pull request merged**